python-lib/ontology_tagging/formatter_by_document.py

```diff
@@ -64,69 +65,60 @@ def _write_row(self, row: pd.Series, language_column: AnyStr = None) -> None:
         """
         language = self._get_document_language(row, language_column)
         document_to_match = self._get_document_to_match(row, language)
-        tags_in_document, keywords_in_document, matched_sentences = [], [], []
+        tags_in_document = []
+        keywords_in_document = []
+        matched_sentences = []
         for idx, sentence in enumerate(document_to_match):
-            (
-                tags_in_document,
-                keywords_in_document,
-                matched_sentences,
-            ) = self._get_tags_in_row(
+            original_sentence = row[self.text_column_tokenized][idx]
+            tags_in_sentence, keywords_in_sentence = self._get_tags_in_sentence(
                 sentence=sentence,
-                original_sentence=row[self.text_column_tokenized][idx],
-                tags_in_document=tags_in_document,
-                keywords_in_document=keywords_in_document,
-                matched_sentences=matched_sentences,
+                original_sentence=original_sentence,
                 language=language,
             )
-
-        if tags_in_document != []:
-            line = {
-                self.tag_columns[0]: self._fill_tags(
-                    True, unique_list(tags_in_document)
-                ),
-                self.tag_columns[1]: self._fill_tags(
-                    True, unique_list(keywords_in_document)
-                ),
-                self.tag_columns[2]: "".join(unique_list(matched_sentences)),
+            tags_in_document.extend(tags_in_sentence)
+            keywords_in_document.extend(keywords_in_sentence)
+            if tags_in_sentence:
+                matched_sentences.append(original_sentence + "\n")
+
+        if tags_in_document:
+            tag_datas = {
```

Comment:
Add small comment here to remind what the elements of `self.tag_columns` are for extra readability

Reply:
Good idea, I added one here 7488672